Gateway/Request/PaymentOffsiteBuilder.php

```diff
@@ -38,26 +35,16 @@ public function __construct(UrlInterface $url)
      */
     public function build(array $buildSubject)
     {
+        $payment = SubjectReader::readPayment($buildSubject);
+        $method  = $payment->getPayment();
+        
         $paymentInfo = [
```

Comment:
@jacekstan once removed below conditions, I think we can simply return this array out. Probably no need to assign to `$paymentInfo` variable anymore?